docs/10_utilities/fill-queue/index.md

```diff
@@ -0,0 +1,118 @@
+---
+content_title: What is Fill-Queue
+link_text: What is Fill-Queue
+---
+
+## What is Fill-Queue
+
+Fill-Queue is an application which accomplishes the following:
+
+1. connects to a state-history enabled `EOSIO` node,
+2. reads the blocks,
+3. filters the desired contracts’ actions and
+4. uses the [AMQ protocol](https://www.amqp.org/) to push messages to a configured queue.
+
+## Overview
+
+Fill-Queue is an application derived from [history-tools](https://developers.eos.io/welcome/v2.1/tools/history-tools/history-tools/). It has to be configured to connect to a state-history enabled `EOSIO` node, from which it reads the blocks, filters the desired contracts’ actions and publishes messages to a queue. Fill-Queue uses the RabbitMQ implementation of the  Advanced Message Queuing Protocol (AMQP).
```

Comment:
we tested Fill-Queue only with RabbitMQ but it is using AMQP which is open standard and not tied to RabbitMQ directly. Maybe it worth to mention that? Something like `Fill-Queue uses Advanced Message Queuing Protocol (AMQP) to connect to Message Queue. Was tested only with RabbitMQ.`

Reply:
yes I think been explicit and clear about the status of the tool is good.